spot_driver/spot_driver/spot_ros2.py

```diff
@@ -40,12 +40,14 @@
 from spot_msgs.srv import SetLocomotion
 from spot_msgs.srv import ClearBehaviorFault
 from spot_msgs.srv import SetVelocity
+from spot_msgs.srv import ExecuteDance
 from spot_msgs.srv import GraphNavUploadGraph, GraphNavClearGraph, GraphNavSetLocalization, GraphNavGetLocalizationPose
 
 #####DEBUG/RELEASE: RELATIVE PATH NOT WORKING IN DEBUG
 # Release
 from .ros_helpers import *
 from spot_wrapper.wrapper import SpotWrapper, CameraSource
+from spot_wrapper.spot_dancing import SpotDancing
```

Comment:
Left a comment in `spot_wrapper`, but is there a reason this needs to  be exposed separately from `SpotWrapper`. As it is just using one more of the various clients from the spot sdk, I would think that it would be contained in the `spot_wrapper` object.

Reply:
Same feedback as @amessing-bdai